api/policies/sessionAuth.js

```diff
@@ -21,10 +21,6 @@ module.exports = function(req, res, next) {
   // User is allowed, proceed to the next policy,
   // or if this is the last policy, the controller
   if (req.user || (req.isAuthenticated && req.isAuthenticated())) {
-    //Redirect mods to the modauth page if they only have normal user scope.
-    if (req.user.isMod && req.session.state.substr(-9) === '_modlogin') {
-      return res.redirect('/auth/modauth');
-    }
```

Comment:
This isn't needed anymore because we now check whether a user is a mod before we log them in.